web/subscribers/index.php

```diff
@@ -1,35 +1,46 @@
 <?php
 
+// File to use the required functions
 require_once("../functions.php");
 
-// Default parameters if not specified
-$defaults = [
-    "color" => "blue",
+// making the youtube Api Url to access the JSON data
+$apiUrl = "https://www.googleapis.com/youtube/v3/channels?";
+$apiUrl .= http_build_query(array(
+            "part" => "statistics",
+            "id"   => "UCipSxT7a3rn81vGLw9lqRkg",// your channel id
+            "alt"  => "json",
+            "key"  => "[YOUR API KEY HERE]" // you can put your API key here if you deploy it yourself
+                            ));
+//getting the data from the url as JSON
+$response = curl_get_contents($apiUrl);
+
+// decoding the json data hence we can extract the Subscribers count
+$decoded = json_decode($response);
+
+// getting the subscribers count by querying throught the decoded json
+$Subscribers = $decoded->items[0]->statistics->subscriberCount;
+
+// getting the subscribers count to a rounded value and right format
+$message = shortNumber($Subscribers);
```

Comment:
This should all be done in the `getShieldsURL` function in `functions.php`. Same with the views.

The `$query` parameter that the function takes can be changed to be a string parameter that takes one of "subscriberCount" or "viewCount" and you can extract either number with

```
$number = $decoded->items[0]->statistics[$query];
```

Reply:
Hey @DenverCoder1, I have made the required changes as you requested, please review them...

Reply:
Hey @DenverCoder1, I have made the required changes as you requested, please review them...